### PR TITLE
bugfix/16087-stocktools-ie

### LINKED ISSUE #16087: Stock-tools error in IE11
#### Behaviour
Error in:
```
// if the button is disabled, don't do anything
                if (buttonWrapperClass.indexOf('highcharts-disabled-btn') > -1) {
```
in stock-tools.js
The error message is about a fail with `indexOf` on undefined.

#### Live demo with steps to reproduce
https://www.highcharts.com/demo/stock/stock-tools-gui

#### Product version
9.1.0+

### PR DESCRIPTION
**Pull request by khlieng**
Fixed #16087, stock tools threw in IE.

### CODE REVIEW

**Review by pawelfus — APPROVED** [review]
Thanks!

File: ts/Extensions/Annotations/NavigationBindings.ts [hunk 1]
```diff
@@ -1602,27 +1602,30 @@ addEvent(Chart, 'render', function (): void {
...
+                                const button = buttonNode[i];
                                 if (button.className.indexOf(disabledClassName) === -1) {
                                     button.className += ' ' + disabledClassName;
                                 }
-                            });
+                            }
                         } else {
-                            buttonNode.forEach(function (button): void {
+                            for (let i = 0; i < buttonNode.length; i++) {
+                                const button = buttonNode[i];
```
Comment (raf18seb) [inline ts/Extensions/Annotations/NavigationBindings.ts:1623]: What about simply convert a nodeList to an array like this?
```js
const buttonNode = Array.prototype.slice.call(
    chart.navigationBindings.container[0].querySelectorAll('.' + key)
);
```

**Review by raf18seb — COMMENTED** [review]
Just a small suggestion

File: ts/Extensions/Annotations/NavigationBindings.ts (hunk 1 above)
Comment (khlieng) [reply ts/Extensions/Annotations/NavigationBindings.ts]: I moved the for loop outside the ifs instead.

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**